File: api/HotelApi/HotelApi/Room.cs
using System;

namespace HotelApi
{
  public class Room
  {
    public int Id { get; set; }
    public string Title { get; set; }
    public string Description { get; set; }
    public decimal ReservationPrice { get; set; }
    public decimal FullPrice { get; set; }
    public DateTime Available { get; set; }
    public double Rating { get; set; }
  }
}

File: src/app/rooms/room.ts
export class Room {
    constructor(rating?: number) {
        this.rating = rating;
    }
    
    public id: number;
    public title: string;
    public description: string;
    public reservationPrice: number;
    public fullPrice: number;
    public available: string;
    public rating: number;
}
